tests/integration/test_integration.py

```diff
@@ -747,19 +748,14 @@ def test_downgrade(encoding, monkeypatch):
 
 @allencodings
 def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
-
-    t = Tracer()
```

Comment:
I think we still need to create this tracer instance, no?

Because we need init/config to happen to pick up on `DD_TRACE_API_VERSION`, otherwise this test is going to test only the default encoding only.